packages/react-components/react-tags-preview/src/utils/types.ts

```diff
@@ -1,3 +1,15 @@
+import * as React from 'react';
+
 export type TagAppearance = 'filled' | 'outline' | 'brand';
 export type TagShape = 'rounded' | 'circular';
 export type TagSize = 'extra-small' | 'small' | 'medium';
+
+export type TagValue = string;
+
+export type TagDismissData<Value = TagValue> = {
+  dismissedTagValue: Value;
```

Comment:
nit:

```suggestion
  value: Value;
```
`dismissedTagValue` sounds a bit verbose, this will already be located in the `onDismiss` event handlers, we can infer by context that the `value` is the dismissed tag value?!

Reply:
Ah true! updated.